public/mathjson.py

```diff
@@ -16,7 +16,7 @@
 _known_functions_mathjson = {
     'acos': 'Arccos',
     'acosh': 'Arcosh',
-    'asin': 'Arcsin',
+    'asin': 'Arsin',
```

Comment:
Did a mistake sneak in here? I think it should be `Arcsin`

Reply:
Yep, you are correct, I must have been looking at the hyperbolic sin. I will correct. this.
https://cortexjs.io/compute-engine/reference/trigonometry/